### PR TITLE
All products view

### PR DESCRIPTION
**Pull request by kesslerjohn**
This is still in progress - I need to create tests and populate the view with products from the database.

### CODE REVIEW

**Comment by kesslerjohn** [comment]
This adds a page where users can view products following the /products path. The products view is really rudimentary - the biggest update that will still need to be made is dynamically populating products from the database. Rspec tests are passing with 318 / 325 LOC (97.85%) covered.

File: config/routes.rb [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
   get "pages/index"
-  resources :users
+
+  get "/signup", to: "users#new", as: "signup"
+  post "/signup", to: "users#create", as: "signup_submit"
 
   get "/login", to: "sessions#new", as: "login"
   post "/login", to: "sessions#create", as: "login_submit"
   get "/logout", to: "sessions#destroy", as: "logout"
+  get "/products", to: "products_view#display", as: "products"
```
Comment (evfinkn) [inline config/routes.rb:11]: I think that—following convention over configuration—this `ProductsViewController#display` can be moved into `ProductsController#index` (and then moving all of the specs, views, etc. to match).

File: app/views/products_view/display.html.erb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  <meta charset="UTF-8" name="viewport" content="width=device-width,initial-scale=1">
+  <%= csrf_meta_tags %>
+  <%= csp_meta_tag %>
+
+  <%= stylesheet_link_tag "application" %>
+  <title>Product Page</title>
+
+</head>
+<body>
+
```
Comment (evfinkn) [inline app/views/products_view/display.html.erb:13]: This view should automatically be rendered inside `layouts/application.html.erb`, so I think this is unneeded.

File: app/views/layouts/.DS_Store
Comment (evfinkn) [inline app/views/layouts/.DS_Store:1]: This is a macOS system file and doesn't need to be committed. We should probably add `.DS_Store` to the `.gitignore`.

**Review by evfinkn — APPROVED** [review]

File: database.yml
[the remark this answers is not in the document]
Comment (kesslerjohn) [reply database.yml:1]: Yep - removed!

File: config/routes.rb (hunk 1 above)
Comment (kesslerjohn) [reply config/routes.rb]: Done 👍

File: app/views/products_view/display.html.erb (hunk 2 above)
Comment (kesslerjohn) [reply app/views/products_view/display.html.erb]: Done 👍

File: app/views/layouts/.DS_Store
Comment (kesslerjohn) [reply app/views/layouts/.DS_Store:1]: Oops, I always forget about this. Added!

**Pull request merged**